rfd/0000-rfds.md

```diff
@@ -101,3 +101,11 @@ The purpose of the `state` is to tell the reader whether they should care about
 this RFD at all. For example, `deprecated` RFDs can be skipped most of the
 time. `implemented` is relevant to Teleport users, but `draft` is mostly for
 Teleport engineers and early adopters.
+
+### Security
+
+Describe the security considerations for your design doc.
```

Comment:
@codingllama would be helpful to give some guidance here:

* Explore possible attack vectors and see how Teleport protects them
* If design if frontend, explore common web vulnerabilities and see if design would prevent them
* Explore possible DDOS and other outage type attacks and see how design considers protecting from them
* If using crypto, show that best practices were considered